### PR TITLE
[Feature]: CSS decoupling

### PR DESCRIPTION
**Pull request by rediris**
Feel free to review/approve, but please don't merge yet... we need to sync this with the theme PR.

Coordinate with staging theme merge: https://github.com/gymnasium/openedx-theme/pull/126/

### CODE REVIEW

**Comment by jgagne** [comment]
gym-dev-prod-courses-nav

gym-dev-prod-about-sidebar-heading-rules

**Comment by jgagne** [comment]
`minor` `cosmetic`

Add more vertical, top and bottom, spacing to Courses page in-page navigation:

https://gymnasium.dev/courses vs https://thegymnasium.com/courses

gym-dev-prod-courses-nav

Update Courses sidebar heading and horizontal rules, see prod About page for reference to use for both pages, Courses and About:

https://gymnasium.dev/courses vs https://thegymnasium.com/courses

gym-dev-prod-courses-sidebar-heading

Prod About page for reference to use for both pages, Courses and About:

https://gymnasium.dev/about vs https://thegymnasium.com/about

gym-dev-prod-about-sidebar-heading-rules

**Comment by rediris** [comment]
Created a tracker issue to address these specific bugs: https://github.com/gymnasium/tracker/issues/271

File: _sass/components/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
+@mixin font-open-sans {
+  @import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,400i,600,700");
```
Comment (jgagne) [inline _sass/components/_typography.scss:2]: What happens if we don't import Open Sans and let it fallback to our default sans-serif stack?

**Review by jgagne — APPROVED** [review]
Awesome work!

File: _sass/components/_typography.scss (hunk 1 above)
Comment (rediris) [reply _sass/components/_typography.scss:2]: Oh... this is just here for reference - the mixin is defined, but I don't believe it's initialized anywhere...

Comment (jgagne) [reply _sass/components/_typography.scss:2]: Gotcha! 😌

**Pull request merged**